code/game/objects/items/storage/secure.dm

```diff
@@ -61,6 +62,20 @@
 	// -> storage/attackby() what with handle insertion, etc
 	return ..()
 
+/obj/item/storage/secure/emag_act(mob/user)
+	user.show_message("<span class='danger'>Now attempting to scramble internal memory.</span>", 1)

```

Comment:
`to_chat(user, "blah blah blah etc.")`

Reply:
Remind me, how do I designate the span class for to_chat?

Reply:
The second arg is just a normal string, so you'd do it like:
`to_chat(user, "<span class='notice'>blah bluh</span>")`